packages/api/src/external/carequality/shared.ts

```diff
@@ -85,9 +86,14 @@ export const cqOrgDetailsSchema = z.object({
   phone: z.string(),
   email: z.string(),
   role: z.enum(["Implementer", "Connection"]),
+  organizationBizType: z.nativeEnum(OrganizationBizType).optional(),
   parentOrgOid: z.string().optional(),
 });
 
+export const cqOrgDetailsOrgBizRequiredSchema = cqOrgDetailsSchema.required({
```

Comment:
don't think we need a separate schema for it, right? Could probably just add the new attribute to the existing schema. AFAIK, `cqOrgDetailsSchema` is only used in two places - here and one other file

Reply:
We use the type `CQOrgDetails` in several place that wont require the orgBizType so this is needed